TestClass.cpp

```diff
@@ -0,0 +1,234 @@
+#include <iostream>
+#include <cmath>
+#include <cassert>  
+
+using namespace std; //use std::cout
+
+#define FILE (__FILE__)
+#define LINE (__LINE__)
+
+// #define LOG_LENGTH (300)
+const int LOG_LENGTH = 300;
+
+void print_log(const char *filename,
+               int         line,
+               const char *message)
+{
+    printf("F:%s L:%d : %s\n", filename, line, message);
+}
+
+class TestClass
+{
+    private:
+        int var1;
+        int var2;
+        // double var3;
```

Comment:
delete the unnecessary (commented out) code

Reply:
ok. I will do another upload WITH all comments and what we talked about, so I have it saved.. and then I'll do another upload with just the code cleaned up.